pkg/commands/util/init.go

```diff
@@ -43,10 +44,18 @@ func GetKubeConfig() string {
 	return filepath.Join(HomeDir(), ".kube", "config")
 }
 
-func IsNamespaceExist(c client.Client, namespace string) bool {
+// DoesNamespaceExist check namespace exist
```

Comment:
I prefer to keep the previous logic as it is. `IsNamespaceExist` or `DoesNamespaceExist`, as a very simple function to check whether the namespace is existed, we don't care the error message a lot. IMHO, returning a true or false is better.

Reply:
How to deal with the error situation, such as a network error may occur, choose to ignore it?

Reply:
+1 for your change. Thanks.